dev/PushNotifications/PushNotificationsLongRunningTask.StartupTask/packages.config

```diff
@@ -1,6 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Build.Tasks.Git" version="1.1.0-beta-21055-01" targetFramework="native" developmentDependency="true" />
-  <package id="Microsoft.SourceLink.Common" version="1.1.0-beta-20204-02" targetFramework="native" developmentDependency="true" />
```

Comment:
Unclear if the deletion of this element is intentional?

Reply:
Hmm not sure how that happened